develop/schemadoc/Automation/InteractivityOptions.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: Automation-InteractivityOptions
+---
+
+# InteractivityOptions simple type
+
+Defines the interactivity options for an Automation script.
+
+## Content Type
+
+|Item|Facet value|Description|
+|--- |--- |--- |
+|***string restriction***|||
+|&nbsp;&nbsp;Enumeration|Auto|Default option. In the background, the script will be read to try to find out if interactivity is needed. This behavior is the same as in DataMiner versions prior to 10.5.8/10.6.0, where the [Interactivity](xref:DMSScript.Interactivity) tag is not yet supported. |
+|&nbsp;&nbsp;Enumeration|Never|The script will never be interactive. It will not show any UI elements.|
+|&nbsp;&nbsp;Enumeration|Optional|The script will be interactive if it needs to be. It will show UI elements if needed.|
+|&nbsp;&nbsp;Enumeration|Always|The script will always be interactive. It will always show UI elements.|
+
+## Remarks
+
+Available from DataMiner 10.5.8/10.6.0 onwards.<!-- RN 42954 -->
```

Comment:
@JorgeFilipeDias, at the moment, RN 42954 is still set to 10.5.9. Will this still be changed, or should we adjust the pull request?